compiler/qsc/src/interpret.rs

```diff
@@ -454,13 +454,57 @@ impl Interpreter {
     fn lower(
         &mut self,
         unit_addition: &qsc_frontend::incremental::Increment,
-    ) -> (Vec<StmtId>, Vec<ExecGraphNode>) {
+    ) -> core::result::Result<(Vec<StmtId>, Vec<ExecGraphNode>), Vec<Error>> {
```

Comment:
Nit: I don't speak enough Rust to understand why, but the other functions in this file seem to use `std::result::Result` for return types. Are these the same type?